### ISSUE TITLE
Fail to find files on windows 11 powershell without config file

### ISSUE THREAD

**Issue by milelo** [oakmac/standard-clojure-style-js#153]
A configuration file seems to be needed before it will check anything. I would expect it to recursively check from the specified dir by default:
```
❯ standard-clj check .\src\
standard-clj check v0.13.0

No files were passed to the "check" command. Please pass a filename, directory, or --include glob pattern.
```
Works if I explicitly specify a file
```
❯ standard-clj check .\src\cube_puzzle\core.cljs
standard-clj check v0.13.0

✗ \src\cube_puzzle\core.cljs [9.83ms]

0 files formatted with Standard Clojure Style
1 file require formatting
Checked 1 file. [28.78ms]
```
It also works if I create a .standard-clj.edn with {:include ["src/**/*.cljs" "src/**/*.clj"]}

**Comment by oakmac** [comment]
Thank you for the report! This is definitely a bug with the CLI tooling. I do not have access to a Windows machine for testing, but I suspect this is probably something simple that I may be able to figure out.

**Comment by oakmac** [comment]
Fixed with [commit #db857ff4](https://github.com/oakmac/standard-clojure-style-js/commit/db857ff413f0a8625c0cd0c975684244d875705e)

**Comment by oakmac** [comment]
I just published v0.14.0 which hopefully contains a fix for this, although I did not test it on Windows :grimacing: 

@milelo - Are you able to confirm if this fixes the issue?

**Comment by milelo** [comment]
This is now fixed in the latest version. Thanks.
